src/OFS/tests/testObjectManager.py

```diff
@@ -528,64 +528,60 @@ def test_empty_string(self):
         self.assertEqual(str(e),
                          "('Empty or invalid id specified', '')")
 
-    def test_unicode(self):
-        # Unicode can only be handled under Python 3, Python 2 needs
-        # bytestrings.
-        if PY2:
-            e = self.assertBadRequest(u'abc☃')
-            self.assertEqual(
-                str(e), "('Empty or invalid id specified', u'abc\\u2603')")
-        else:
-            # Does not raise
-            self._callFUT(self._makeContainer(), u'abc☃')
+    @unittest.skipIf(PY3, 'Python 3 cannot handle bytestrings here.')
```

Comment:
A minor thing I also have been warned about... it is better to use "not PY2" which is more future proof, and we know we have a long future :)

Reply:
+1 maybe `@unittest.skipUnless(PY2 ...`could be used alternatively to prevent to use the `not`.

Reply:
I changed this to `skipUnless`. Thanks.